python/ray/autoscaler/v2/schema.py

```diff
@@ -19,6 +20,18 @@
 NodeType = str
 
 
+# e.g. head node or worker node.
+class NodeKind(Enum):
+
+    # Head node kind
+    HEAD = NODE_KIND_HEAD
+    # Worker node kind
+    WORKER = NODE_KIND_WORKER
+    # Unmanaged node kind (e.g. user started cloud instance or non
+    # ray node, this happens in KubeRay)
```

Comment:
what's non ray node?

Reply:
changed comments - non-ray node should not be returned by node provider.